x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group3/test_helpers.ts

```diff
@@ -225,22 +225,21 @@ export const expectActionsFired = async ({
   retry: RetryService;
   expectedNumberOfActions: number;
 }) => {
-  const events = await retry.try(async () => {
+  await retry.try(async () => {
```

Comment:
Moved the retry here so that we retry testing that the number of action events match the expected since sometimes it may take actions a little bit to run.